ntbs-ui-tests/Features/CreateNotifications.feature

```diff
@@ -0,0 +1,54 @@
+Feature: Notification creation

```

Comment:
Assuming while doing this you had to fix a couple bits. What does a test failure look like? Is it chucked into console in a readable fashion?

Reply:
Just saw your top level comment about dockerising!

Reply:
Yeah all the errors go to console like for integration/unit tests